providers/digitalocean/create.go

```diff
@@ -105,7 +105,6 @@ func CreateDigitaloceanCluster(definition *pkgtypes.ClusterDefinition) error {
 	secretsFilesToRestore, err := os.ReadDir(ctrl.ProviderConfig.SSLBackupDir + "/secrets")
 	if err != nil {
 		log.Info().Msg(err.Error())
-		return fmt.Errorf("error reading secrets directory for TLS: %w", err)
```

Comment:
Are you sure this should swallow the error?

Reply:
yess i tested,it works :)

Reply:
Can you explain the reason why the error is ignored, please? I'm always very nervous when I see an ignored error.

Reply:
the secrets folder is created when we do backup cmd and not when we provision it,i think we can fully remove the secrets check from create.go of all cloud

Reply:
@mrsimonemms I think it's fine to not return the error. Not every setup uses SSL with certs managed by the cluster (for example, if you use cloudflare) so this could fail on those situations but it isn't really a failure. Good catch though!